packages/functionals/botpress-nlu/src/providers/dialogflow.js

```diff
@@ -10,6 +9,7 @@ export default class DialogflowProvider extends Provider {
 
     this.projectId = this.config.googleProjectId
 
+    const dialogflow = eval("require('dialogflow')") // eslint-disable-line no-eval
```

Comment:
Ideally I'de like to see normal `require` here and drop `webpack` for the node part (as I did for the core package), but it's defo fine to have it like this for now.
Can you add a comment explaining `eval` is needed only because of webpack?

Reply:
I agree, dropping webpack for node-part should be done for all the packages.
Added appropriate comment.